cfgov/scripts/pages_to_redirect.py

```diff
@@ -0,0 +1,46 @@
+import csv
+import logging
+
+from wagtail.core.models import Page
+
+
+logger = logging.getLogger(__name__)
+
+
+# This script is for use on September 30, 2020, when we'll be migrating
+# cf.gov to a new IA. Delete this script after the migration is done.
+# Run this from the command line with this:
+#   cfgov/manage.py runscript pages_to_redirect --script-args [PATH]
+def run(*args):
+    if not args:
+        logger.error("error. Use --script-args [PATH] to specify the " +
+                     "location of the csv.")
+    else:
+        page_moves_file = args[0]
+        redirects_file = 'redirects_list.csv'
+
+        pages_to_move = set()
```

Comment:
I pushed an update in 30ef08b where I order the set by ID after all the entries are added. Do you think that's an okay approach?

Reply:
Thanks -- I was sort of thinking it'd be nice to sort by URL so that the file is more easily viewable, and comparable to the source spreadsheet, but don't feel that strongly since we're only talking about < 40 URLs.

Reply:
Good point that alphabetical order would group so children of the same page would be together. Updated to use that ordering in 4d77d4a.